### PR TITLE
at.cpp: added 286 motherboards and BIOS versions

### PR DESCRIPTION
**Pull request by rfka01**

### CODE REVIEW

File: src/mame/drivers/at.cpp [hunk 1]
```diff
@@ -1709,15 +1685,146 @@ ROM_START ( mkp286 )
...
+	ROMX_LOAD( "biostar mb-1212c.bin", 0x10000, 0x10000, CRC(153a783a) SHA1(acad4a3ffe93d3884dcb743c32d6317a132cda7b), ROM_BIOS(0) )
+	// 1: CHIPS SCAT BIOS Version 125D - MB-1212C
+	ROM_SYSTEM_BIOS(1, "mb1212_2", "mb1212_2")
+	ROMX_LOAD( "mb-1212c.bin", 0x10000, 0x10000, CRC(4675530a) SHA1(c34b1c67ac29695e565363f484e17ab5f8ddaad5), ROM_BIOS(1) )
+ROM_END
+
+
+// ***** Motherboards using the original Chips chipset: P82C202, P82C201, P82A203, P82A204, P82A205
+
+ROM_START( chips0 )
```
Comment (ajrhacker) [inline src/mame/drivers/at.cpp]: CS8220 is the name Chips & Technologies used for their first-generation 82C201-82A205 CHIPset. I'm not sure it makes sense to separate these out, given that these five chips integrate almost no I/O registers whatsoever. (That would happen in the 82C206 Integrated Peripherals Controller, which was used with both the first-generation and "New Enhanced AT" CHIPsets.)

File: src/mame/drivers/at.cpp [hunk 2]
```diff
@@ -2588,12 +2711,20 @@ COMP( 1994, ficvipio,  ibm5170, 0,       at486,     0,     at_state,     init_at
 COMP( 199?, ficvipio2, ibm5170, 0,       at486,     0,     at_state,     init_at,        "First International Computer", "486-VIP-IO2", MACHINE_NOT_WORKING )
 COMP( 199?, hot409,    ibm5170, 0,       at486,     0,     at_state,     init_at,        "Shuttle Computer International", "HOT-409", MACHINE_NOT_WORKING )
 COMP( 198?, aubam12s2, ibm5170, 0,       atvga,     0,     at_state,     init_at,        "AUVA COMPUTER, INC.", "BAM/12-S2", MACHINE_NOT_WORKING )
+COMP( 198?, wyse286,   ibm5170, 0,       ibm5162,   0,     at_state,     init_at,        "Wyse", "WYSEpc", MACHINE_NOT_WORKING )
```
Comment (ajrhacker) [inline src/mame/drivers/at.cpp]: This machine should have a more specific full name. The first WYSEpc (WY-1100) was a mostly-compatible XT clone.

Comment (rfka01) [reply src/mame/drivers/at.cpp]: Right, the WYSEpc one is no problem, I'll give it a better name.
Now how about CS8220 stuff ... do you want them in the "at" section or the "atvga" section?

How about those doublettes between at and atvga. Vas argued that they should be kept as they would be faster 286 machines with a VGA card as standard. Shouldn't they then be deleted from the "at" section?

**Comment by rfka01** [comment]
I've added the CS8220 files to at and atvga per your suggestion on the Shoutbox.

**Pull request merged**